integ/s3/validate-and-clean.go

```diff
@@ -142,6 +142,10 @@ func validate(s3Client *s3.S3, response *s3.ListObjectsV2Output, bucket string,
 				exitErrorf("[TEST FAILURE] Json Unmarshal Error:", decodeError)
 			}
 
+			// On Windows, we would have additional \r which needs to be stripped.
+			// This would be NOOP on Linux.
+			message.Log = strings.ReplaceAll(message.Log, "\r", "")
```

Comment:
To make the comment more clear you could `if runtime.GOOS() == "windows"` then do the strip